Documentation/filters-strings.md

```diff
@@ -166,7 +166,9 @@ This filter accepts a parameter (boolean, default `false`) that controls the pre
 
 ## Filter: `swiftIdentifier`
 
-Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
+This filter has a couple of modes that you can specifiy using an optional argument (defaults to "normal"):
+
+**normal**: Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
 
 - Uppercase the first character.
```

Comment:
Well it's debatable. The aim of `swiftIdentifier` is only to make the string valid Swift. Prettifying the string is nice too, but transforming *too much* means that if we don't want that transformation we have to apply the reverse of that transform afterwards to cancel it… And that reverse transform may not always exist.

Just feels odd to over-transform when half of the cases would need it, sure, but the other half doesn't and would need to revert it, so why force to do it in the first place instead of letting it be opt-in is what I mean. The history of this comes back from Swift 2 of course, but we're passed that now.

But indeed that would be a breaking change so maybe let's keep that for a later PR :-/